rpc_util_test.go

```diff
@@ -290,3 +293,129 @@ func BenchmarkGZIPCompressor512KiB(b *testing.B) {
 func BenchmarkGZIPCompressor1MiB(b *testing.B) {
 	bmCompressor(b, 1024*1024, NewGZIPCompressor())
 }
+
+type testCompressor struct {
```

Comment:
this testCompressor should not be used ideally. If at all we are using it, we should not use it for test cases involving max receive size and actual receive message size

Reply:
done